tic_tac_toe/src/board_test.py

```diff
@@ -0,0 +1,229 @@
+from board import Board
+
+
+def update_column_1():
+    board = Board(3)
+    board.values[board.convert(0, 0)] = "X"
+    board.values[board.convert(1, 0)] = "X"
+    board.values[board.convert(2, 0)] = "X"
+    return board
+
+
+def update_column_2():
+    board = Board(3)
+    board.values[board.convert(0, 0)] = "X"
+    board.values[board.convert(1, 0)] = None
+    board.values[board.convert(2, 0)] = "O"
+    return board
+
+
+def update_column_3():
+    board = Board(3)
+    board.values[board.convert(0, 0)] = None
+    board.values[board.convert(1, 0)] = None
+    board.values[board.convert(2, 0)] = None
+    return board
+
+
+def update_column_4():
+    board = Board(3)
+    board.values[board.convert(0, 0)] = "X"
+    board.values[board.convert(1, 0)] = "O"
+    board.values[board.convert(2, 0)] = "O"
+    return board
+
+
+def update_diagonal_1():
+    board = Board(3)
+    board.values[board.convert(0, 0)] = "X"
+    board.values[board.convert(1, 1)] = "X"
+    board.values[board.convert(2, 2)] = "X"
+    return board
+
+
+def update_diagonal_2():
+    board = Board(3)
+    board.values[board.convert(0, 0)] = "X"
+    board.values[board.convert(1, 1)] = None
+    board.values[board.convert(2, 2)] = "O"
+    return board
+
+
+def update_diagonal_3():
+    board = Board(3)
+    board.values[board.convert(0, 0)] = "X"
+    board.values[board.convert(1, 1)] = "O"
+    board.values[board.convert(2, 2)] = "X"
+    return board
+
+
+def update_diagonal_4():
+    board = Board(3)
+    board.values[board.convert(0, 0)] = None
+    board.values[board.convert(1, 1)] = None
+    board.values[board.convert(2, 2)] = None
+    return board
+
+
+def update_diagonal_5():
+    board = Board(3)
+    board.values[board.convert(0, 2)] = "X"
+    board.values[board.convert(1, 1)] = "X"
+    board.values[board.convert(2, 0)] = "X"
+    return board
+
+
+def update_row_1():
+    board = Board(3)
+    board.values[board.convert(0, 0)] = "X"
+    board.values[board.convert(0, 1)] = "X"
+    board.values[board.convert(0, 2)] = "X"
+    return board
+
+
+def update_row_2():
+    board = Board(3)
+    board.values[board.convert(0, 0)] = "X"
+    board.values[board.convert(0, 1)] = None
+    board.values[board.convert(0, 2)] = "O"
+    return board
+
+
+def update_row_3():
+    board = Board(3)
+    board.values[board.convert(0, 0)] = None
+    board.values[board.convert(0, 1)] = None
+    board.values[board.convert(0, 2)] = None
+    return board
+
+
+def update_row_4():
+    board = Board(3)
+    board.values[board.convert(0, 0)] = "X"
+    board.values[board.convert(0, 1)] = "O"
+    board.values[board.convert(0, 2)] = "O"
+    return board
+
+
+def validate_move_update_1():
+    board = Board(3)
+    board.values[0] = "X"
+    return board
+
+
+def test_column_1():
```

Comment:
Describe the situation that you're testing in a string and name the test function accordingly.  Also, move the implementation of the update function into here.  I shouldn't have to bounce between the top of the file down to here to figure out what situation you're testing.